src/components/ButtonComponent/ButtonComponent.css

```diff
@@ -4,13 +4,17 @@
  background-color: #F7B21F;
  border-radius: 50%;
  border: none;
- box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+ box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
  cursor: pointer;
 }
 
 
-.RoundAddButton:focus {
+.RoundAddButton:hover {
     box-shadow: none;
     border-color:transparent!important;
     outline:none;
 }
+
+.RoundAddButton svg{
```

Comment:
simply make `.RoundAddButton` a flex item and set `justify-content` and `align-items` to `center`

Reply:
...that and remove the !important

Reply:
@StephenAraka @mrwagaba this has been worked on.